src/fastoad_cs25/models/geometry/geom_components/vt/components/compute_vt_mac.py

```diff
@@ -60,15 +60,17 @@ def compute(self, inputs, outputs):
         sweep_25_vt = inputs["data:geometry:vertical_tail:sweep_25"]
         b_v = inputs["data:geometry:vertical_tail:span"]
 
-        tmp = root_chord * 0.25 + b_v * math.tan(sweep_25_vt / 180.0 * math.pi) - tip_chord * 0.25
-
         mac_vt = (
             (root_chord**2 + root_chord * tip_chord + tip_chord**2)
             / (tip_chord + root_chord)
             * 2.0
             / 3.0
         )
-        x0_vt = (tmp * (root_chord + 2 * tip_chord)) / (3 * (root_chord + tip_chord))
+
+        x0_vt = (root_chord / 4) + (b_v / 3) * math.tan(sweep_25_vt / 180.0 * math.pi) * (
```

Comment:
Same remark as above.

Reply:
Corrected in [a81f76f](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/45/commits/a81f76ffb5eedb442f8fc08a514ad1d8795daf43)